jobs/config.json

```diff
@@ -11122,6 +11122,7 @@
   "ci-kubernetes-kubemark-5-gce": {
     "args": [
       "--cluster=kubemark-5",
+      "--env=KUBE_GCE_ENABLE_IP_ALIASES=true",
```

Comment:
Please put it in .env file.

Reply:
We will forget about it.
In that case, I would even prefer having making a dedicated copy of .env file for that purpose.

Reply:
I changed it to kubemark-100 - PTAL.